qemu/tests/cfg/remote_image_convert.cfg

```diff
@@ -95,5 +95,20 @@
             # target image is 'convert', converted from remote
             iscsi_direct:
                 lun_convert = 1
-            nbd:
-                skip_target_creation_convert = yes
```

Comment:
since the produce bug has been fixed, no need to set skip_target_creation_remote. Let's remove related lines, keep the same with other network backends